content/03.results.md

```diff
@@ -177,36 +177,47 @@ This trend may be the result of therapy-induced mutations which produce addition
-With one exception, hypermutant and ultra-hypermutant tumors had high _TP53_ scores and telomerase activity **Figure {@fig:Fig4}D**.
-Interestingly, none of the samples were exposed to signatures 3 (present in homologous recombination deficient tumors), signature 8 (arises from double nucleotide substitutions/unknown etiology), or signature N6 (a universal CNS tumor signature).
-The mutual exclusivity of signatures 3 and MMR corroborates a previous report suggesting tumors do not have both deficient homologous repair and mismatch repair [@doi:10.1016/j.celrep.2018.03.076].
+We further investigated the mutational signature profiles of the hypermutant (TMB > 10 Mut/Mb; N = 2) and ultra-hypermutant (TMB > 100 Mut/Mb; N = 4) tumors and/or derived cell lines, all of which were HGGs, from five patients in the OpenPBTA cohort (**Figure {@fig:Fig4}E**).
+Signature 11, which has been suggested to result from exposure to temozolomide, was indeed present in tumors with previous exposure to the drug (**Table 2**).
+The MMR2 signature was present in tumors of four patients (PT_0SPKM4S8, PT_3CHB9PK5, PT_JNEV57VK, and PT_VTM2STE3) diagnosed with Lynch syndrome (**Table 2**), a genetic predisposition syndrome caused by a variant in a mismatch repair gene such as _PMS2_, _MLH1_, _MSH2_, _MSH6_, or others [@doi:10.3390/genes5030497].
+Interestingly, while the cell line derived from patient PT_VTM2STE3's tumor at progression was not hypermutated (TMB = 5.74 Mut/Mb), it only showed exposure the MMR2 signature, suggesting selective pressure to maintain a mismatch repair (MMR) phenotype _in vitro_.
+From patient PT_JNEV57VK, only one of the two cell lines derived from the progressive tumor was hypermutated (TMB = 35.9 Mut/Mb).
+This hypermutated cell line had a strong exposure to signature 11, while this patient's non-hypermutated cell line showed a number of lesser exposures (1, 11, 18, 19, MMR2), highlighting the plasticity of mutational processes and the need to carefully genomically characterize and select models for preclinical studies based on research objectives.
+We observed that signature 18, which has been associated with high genomic instability and can lead to a hypermutator phenotype [@doi:10.1038/s43018-020-0027-5], was uniformly represented among hypermutant solid tumors. 
+Additionally, we found that all of these tumor or cell lines had dysfunctional _TP53_ (**Table 2**), consistent with a previous report showing _TP53_ dysregulation is a dependency of tumors with high genomic instability [@doi:10.1038/s43018-020-0027-5].
+With one exception, hypermutant and ultra-hypermutant tumors had high  _TP53_ scores (> 0.5) and telomerase activity **Figure {@fig:Fig4}D**. **TODO: 1/3 of the telomerase activities are <0.5, so what counts as "high" here?**
```

Comment:
> The lowest they used was 0.5

I'm not sure what this means, can you clarify? Who is they?


We might want to stick to just tp53 here without any cutoff for telomerase.

Reply:
This was in reference to the authors of the EXTEND manuscript - they didn't state exactly what a high or low score was either, but used distributions for cutoffs, so we can just remove. That's fine. I was just trying to draw attention to the colored dots in the telomerase panel of the boxplot as well since we otherwise don't discuss, but maybe we can just skip here since the colored dots are noted in the legend.